Animations/Zoom/LeftBig.js

```diff
@@ -26,5 +28,5 @@ var ZoomLeftBig = {
     'transform': 'scale(.1) translateX(-2000px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('ZoomLeftBig', ZoomLeftBig);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('ZoomLeftBig', ZoomLeftBig);
```

Comment:
This does not need to have `'ZoomLeftBig'` passed into it since we have defined a `className` property in `ZoomLeftBig`.